department-of-veterans-affairs/va.gov-cms
# VACMS-3294: Removed file entity embed too from rich text editor.

## Description

See _issueid_. #3294 

## Testing done


## Screenshots


## QA steps

- Navigate to preview environment:
- Navigate to the following node: `/node/286/edit`
- Select the `Add Rich Text` button
- [ ] Confirm that the File Entity Embed is not an option in the active rich text toolbar.
![image](https://user-images.githubusercontent.com/56809719/114597583-f0752500-9c5e-11eb-9242-657a84f509a9.png)


### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
